lib/generator.js

```diff
@@ -1,38 +1,159 @@
 'use strict';
 
+const common = require('metarhia-common');
+
+const structures = [
+  'boolean',
+  'null',
+  'undefined',
+  'number',
+  'string',
+  'symbol',
+];
+
+const gObjects = [
```

Comment:
What is `gObjects` and why not just `objects`?

Edit: I assume this is due to the naming in MDN though I think it'd be better to use full name `globalObjects`.